picard/releasegroup.py

```diff
@@ -33,6 +33,9 @@
 from picard.util import uniqify
 
 
+MAX_TRACKS = 10
```

Comment:
MAX_MEDIA_DISCS? Not tracks anyway.

Should this be in picard/picard/const/__init__.py ?

Reply:
It's displayed in "Tracks" field hence the name.
And no, it should stay here as it's very specific to the code in this file. I'd perhaps even make it more local.

Reply:
Local = OK. You know best.